content/docs/user-guide/assets/asset-processor/configuration.md

```diff
@@ -5,21 +5,77 @@ description: Configure Asset Processor for Open 3D Engine (O3DE) with AssetProce
 weight: 300
 toc: true
 ---
+
+### Settings Registry
 
-You can modify the engine settings registry file for **Asset Processor** in `Registry/AssetProcessorPlatformConfig.setreg` to perform the following tasks:
+Many of the content processing rules that **Asset Processor** uses for your project are controlled by settings registry values. You can view the default values in [`Registry/AssetProcessorPlatformConfig.setreg`](https://github.com/o3de/o3de/blob/development/Registry/AssetProcessorPlatformConfig.setreg). This file is well commented, and full of examples you can use for specific configuration on your project's asset pipeline.
 
-* Add new source asset types for Asset Processor.
-* Update existing file type rules.
-* Update the ignore list.
-* Add or remove target platforms.
+{{< todo issue="https://github.com/o3de/o3de.org/issues/706" >}}
+To add project-specific or user-specific overrides, you can use the same pattern you would follow with other settings registry overrides. The linked ticket covers documenting the setttings registry override functionality.
+{{</todo>}}
 
-    {{< note >}}
-Asset Processor automatically builds assets for the host platform. For example, if Asset Processor is running on Windows, Asset Processor builds Windows assets even if pc is not enabled in the `.setreg` file.
-    {{< /note >}}
+### Processing Assets for Multiple Platforms
+
+To process assets for a non-host platform, you can enable additional platforms under the settings registry path `Amazon/AssetProcessor/Settings/Platforms`. Note that this is only one step in enabling a platform for your project. See the [platforms](/docs/user-guide/platforms/) section of the documentation for how to set up and deploy to additional platforms.
+
+### Asset Job Management
+
+By default, the Asset Processor will create a worker job for each logical core on the host machine. If you wish to manage this yourself, you can adjust the setting `Amazon/AssetProcessor/Settings/Jobs/maxJobs`. A value of zero will have the Asset Processor create a job for each logical core on your machine. A value of one can be useful when debugging, if you wish to attach a debugger to a running builder processing a specific asset. Other values lower than your logical core count can be useful to manage the impact asset processing has on your machine while you are trying to perform other tasks, with the tradeoff of a longer time before all assets are processed.
+
+### Asset Cache Server
+
+The Asset Cache server's address can be set using the setting path `Amazon/AssetProcessor/Settings/Server/CacheServerAddress`. In most cases this will be a network share, an example for Windows looks like `T:\\AssetServerCache`. This feature is in development. If you have questions about the Asset Cache server, reach out to us on the [O3DE Discord](https://discord.com/invite/o3de).

```

Comment:
Do we have a Linux example ie samba share or similar? Maybe add explicit {{note}} that this feature is in development and that users should ask questions in sig-content?

Reply:
I think it only supports Windows